src/utils/hints/nftHints.js

```diff
@@ -7,4 +7,9 @@ export const nftsHints = {
   extensions: 'Extensions that are implemented for this NFT according to the creator of the smart contract.',
   tokenLimit: 'The maximum number of NFTs that can be minted.',
   templateLimit: 'The maximum number of NFTs that can be minted per template.',
+  transferHash: 'The hash of the transaction that transfers the NFT.',
```

Comment:
```suggestion
  transferHash: 'The hash of the transaction that transferred the NFT.',
```

I would keep the same verb tense of the token id hint.